rotkehlchen/globaldb/handler.py

```diff
@@ -299,24 +299,31 @@ def get_assets_mappings(identifiers: List[str]) -> Dict[str, dict]:
         Given a list of asset identifiers, return a list of asset information(id, name, symbol)
         for those identifiers.
         May raise:
-        - InputError if one of the identifiers do not exist
+        - InputError if one of the identifiers do not exist or invalid asset type found.
         """
         result = {}
         identifiers_query = f'assets.identifier IN ({",".join("?" * len(identifiers))})'
-        with GlobalDBHandler().conn.read_ctx() as cursor:
-            cursor.execute(
-                'SELECT assets.identifier, name, symbol, chain FROM assets '
-                'LEFT JOIN common_asset_details on assets.identifier = common_asset_details.identifier '  # noqa: E501
-                'LEFT JOIN evm_tokens ON evm_tokens.identifier= assets.identifier '
-                'WHERE ' + identifiers_query,
-                tuple(identifiers),
-            )
-            for entry in cursor:
-                result[entry[0]] = {'name': entry[1], 'symbol': entry[2]}
-                if entry[3] is not None:
-                    result[entry[0]].update({'evm_chain': ChainID.deserialize_from_db(entry[3]).serialize()})  # noqa: E501
-            if len(result) != len(identifiers):
-                raise InputError('One or more of the given identifiers could not be found in the database')  # noqa: E501
+        try:
```

Comment:
Why is this try except needed?

Reply:
`AssetType.deserialize_from_db()` raises DeserializationError

Reply:
But isn't it guaranteed to always exist due to a foreign key? https://github.com/rotki/rotki/blob/develop/rotkehlchen/globaldb/schema.py#L115

Reply:
Yeah yeah. Updated.